src/main/java/com/newrelic/graphql/schema/SimpleGraphQLBuilder.java

```diff
@@ -98,8 +98,7 @@ public GraphQL build() {
     }
 
     SchemaGenerator schemaGenerator = new SchemaGenerator();
-    SchemaGenerator.Options options =
-        SchemaGenerator.Options.defaultOptions().enforceSchemaDirectives(false);
+    SchemaGenerator.Options options = SchemaGenerator.Options.defaultOptions();
```

Comment:
Suspect this change is going to have some impact for users of our library since we have some internal usage of a `@java` directive (discussion [here](https://github.com/graphql-java/graphql-java/pull/1773)) Will test it out and follow up separately if there are issues.

Reply:
Sounds good. Thanks! 👍